src/tests/ftest/vmd/fault_reintegration.py

```diff
@@ -103,8 +103,7 @@ def test_nvme_fault_reintegration(self):
             9. Replace the same drive back.
             10. Drive status LED should be off indicating good device is plugged-in.
 
-        :avocado: tags=all,daily_regression
-        :avocado: tags=hw,medium
+        :avocado: tags=manual
```

Comment:
We should keep the stage tags just in case this test ever gets ran in CI. Otherwise, CI assumes it is a `vm` test
```suggestion
        :avocado: tags=manual
        :avocado: tags=hw,medium
```

Or, if we expect this test to NEVER be ran in CI, we should remove the feature tags `vmd,vmd_led`

With these current tags, if a PR runs:
`Test-tag: vmd`
Then CI WILL try to run this test on VM.